pom.xml

```diff
@@ -239,7 +239,13 @@ limitations under the License.
                 </requireSameVersions>
                 <requireSameVersions>
                   <dependencies>
-                    <dependency>io.opencensus:*</dependency>
+                    <!-- gax-grpc transitvely brings in opencensus-proto, which the latest version is 0.2.0-->
+                    <!-- Explicitly list the opencensus deps so we can allow both 0.2.0 and latest for other modules-->
+                    <dependency>io.opencensus:opencensus-api</dependency>
+                    <dependency>io.opencensus:opencensus-impl</dependency>
+                    <dependency>io.opencensus:opencensus-contrib-zpages</dependency>
+                    <dependency>io.opencensus:opencensus-exporter-stats-stackdriver</dependency>
+                    <dependency>io.opencensus:opencensus-exporter-trace-stackdriver</dependency>
```

Comment:
consider banning other opencensus dependencies to ensure we dont miss one sneaking in:

https://maven.apache.org/enforcer/enforcer-rules/bannedDependencies.html

<bannedDependencies>
  <excludes>
     <exclude> io.opencensus:opencensus-api</exclude>
     .. everything you listed + the -proto
  </excludes>
  <includes>
     <opencensu:*

Reply:
@igorbernstein2 updated, wdyt?